bridge_adaptivity/module/forms.py

```diff
@@ -153,9 +153,14 @@ class Meta:
             'grading_policy_name',
             'strict_forward',
             'ui_option',
-            'ui_next'
+            'ui_next',
+            'congratulation_message',
         )
-        labels = {'ui_option': _('UI Option'), 'ui_next': _('Additional NEXT Button')}
+        labels = {
+            'ui_option': _('UI Option'),
+            'ui_next': _('Additional NEXT Button'),
+            'congratulation_message': _('Show congratulation message if score is 95%+')
```

Comment:
If our congrats param is updated in the settings, the message is incorrect

Reply:
I changed it. I used the settings for showing the correct level.